Build fails for .NET SDK when solution file doesn't exist in the package directory

### Prompts Used

Build the Azure.Communication.Messages SDK project. Can you build current sdk project?

### Language

.NET

### Service or Resource Provider

Azure.Communication.Messages

### Logs, Traces, or Screenshots

The build failed because there's no `.sln` file in this directory. Let me check for a `.csproj` file and build that directly:

<img alt="Build error screenshot" width="816" src="https://github.com/user-attachments/assets/b7800173-fb5c-4ac6-8a46-88017576dd32" />

### Feedback

There are some dotnet SDK projects without a solution file (.sln) in the package directory. In this case, the build tool runs `dotnet build` at the package directory, which fails if no solution or project file is present. Example error: 'MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.' Please handle this scenario in the build tool.

### Solution

We might run the `dotnet build` at the `package-dir\src` directory.

--- a/eng/swagger_to_sdk_config.json
+++ b/eng/swagger_to_sdk_config.json
@@ -23,7 +23,7 @@
   "packageOptions": {
     "packageFolderFromFileSearch": false,
     "buildScript": {
-      "command": "dotnet build {packagePath}"
+      "command": "dotnet build {packagePath}/src"
     },
     "updateMetadataScript": {
       "path": "./eng/scripts/Export-API.ps1"

